querybook/webapp/components/QueryPeerReviewModal/QueryPeerReviewModal.tsx

```diff
@@ -126,7 +129,7 @@ const QueryPeerReviewForm: React.FC<IQueryPeerReviewFormProps> = ({
                             className="mb12"
                             type="warning"
                             size="small"
-                            message="Tip: Before submitting your sensitive query for review, run it once to validate its syntax and feasibility. This helps identify errors early and reduces unnecessary review cycles."
+                            message={reviewTip}
```

Comment:
I guess the review tip could be optional. only render the message banner when provided.